### DISCUSSION TITLE
user event settings Admin UI

### DISCUSSION THREAD

**Discussion by koschib** [keycloak/keycloak#23412] [Q&A]
Subject: Admin Console Keycloak Server 

In the realm settings at events/user events it is not possible to deactivate the saving again. It shows it grayed out when deactivating it but does not save the deactivation. As soon as I reload the page the saving of events is active again.

Does anyone have any ideas?

![Bildschirmfoto 2023-09-21 um 08 13 55](https://github.com/keycloak/keycloak/assets/98027016/75d9e4d7-12fb-443b-88d1-e55cb7cff548)

**Comment by jonkoops** [comment]
Hi @koschib, if this is a reproducible problem in the latest version of Keycloak please log an issue for it.
